cfgov/unprocessed/css/on-demand/pagination.less

```diff
@@ -0,0 +1,40 @@
+/* ==========================================================================
+   cfgov-refresh
+   Master Less File.
+   ========================================================================== */
+
+
+/* Base styles
+   @import combined CF components Less file and make any overrides.
+   ========================================================================== */
+
+// Import required CF components.
+// Combined CF components.
+@import (less) 'cf-core/src/cf-core.less';
+@import (less) 'cf-icons/src/cf-icons.less';
+@import (less) 'cf-buttons/src/cf-buttons.less';
+@import (reference) 'cf-layout/src/cf-layout.less';
+
+// The Capital Framework (CF) brand color palette.
+@import (reference) "../brand-palette.less";
+
+// Override the CF theme variables with colors from the brand color palette.
+@import (reference) "../cf-theme-overrides.less";
+
+// Enhancements that are on a migration path to being added to CF.
+// CF Core
+@import (reference) "../enhancements/vars.less";
+@import (reference) "../enhancements/media-queries.less";
+@import (less) "../enhancements/base.less";
+@import (less) "../enhancements/layout.less";
+
+
```

Comment:
We can't unfortunately guarantee that a component will appear with another component (like a Salesforce app may use the header/footer, but a custom Salesforce component that looks like pagination but isn't). It might work better to see if we can get a css redundancy remover to work. When I tried that though it mangled things.